platform/openide.util.ui/src/org/openide/util/Utilities.java

```diff
@@ -790,6 +790,13 @@ private static synchronized NamesAndValues initNameAndValues() {
         values.put(0x290,"MOUSE_WHEEL_UP");
         values.put(0x291,"MOUSE_WHEEL_DOWN");
 
+        for (int button = 4; button < 10; button++) {
+            String name = "MOUSE_BUTTON" + button; // NOI18N
+            int code = 0x292 + (button - 1);
+            names.put(name, code);
```

Comment:
It's my understanding that check was added because of those keys being added to the JDK?  If anything we have the opposite problem and should check if the keycode is defined, but that applies to the existing mouse wheel pseudocodes already.